pages/vulnerabilities/Insufficient_Session-ID_Length.md

```diff
@@ -13,33 +13,124 @@ tags: vulnerability, Insufficient Session-ID Length
 
 ## Description
 
-Session identifiers should be at least 128 bits long to prevent brute-force session guessing attacks.
+Session identifiers must have **at least 64 bits of entropy** to prevent brute-force session guessing attacks. It's important to understand that session ID length alone is not a sufficient indicator of security. Instead, the critical factor is the entropy (or randomness) contained within the session identifier. While the length of the session ID can affect how much entropy is encoded, the actual entropy depends on how the session ID is generated. Encoding methods (e.g., hexadecimal, Base64, etc) influence how the session id is represented, but the security ultimately comes from the randomness of the identifier, not the encoding itself.
```

Comment:
Could you ensure consistency in ID vs id vs Id?

This paragraph uses one and two. Chances are there are other places in the doc too.

IMHO full caps is the way to go, but consistency is king.